nekoyume/Assets/_Scripts/BlockChain/RPCAgent.cs

```diff
@@ -327,30 +328,14 @@ private async Task<long> GetNonceAsync()
 
         public void OnRender(byte[] evaluation)
         {
-            var formatter = new BinaryFormatter();
-            using (var compressed = new MemoryStream(evaluation))
-            using (var decompressed = new MemoryStream())
-            using (var df = new DeflateStream(compressed, CompressionMode.Decompress))
-            {
-                df.CopyTo(decompressed);
-                decompressed.Seek(0, SeekOrigin.Begin);
-                var ev = (ActionEvaluation<ActionBase>)formatter.Deserialize(decompressed);
-                ActionRenderer.ActionRenderSubject.OnNext(ev);
-            }
+            var ev = MessagePackSerializer.Deserialize<NCActionEvaluation>(evaluation).ToActionEvaluation();
```

Comment:
It's different from `var ev = MessagePackSerializer.Deserialize<ActionEvaluation<ActionBase>>(evaluation);` in OnUnrender(), what's the reason?

Reply:
i missed this point. i'll fix `OnUnrender` too.